scarlet/display.py

```diff
@@ -308,15 +314,23 @@ def show_scene(
     linear: bool
         Whether or not to display the scene in a single line (`True`) or
         on multiple lines (`False`).
+    use_flux: bool
+        Whether to show the flux redistributed model (`source.flux`) or
+        the model itself (`source.get_model()`) for each source.
```

Comment:
This only works for a `lite` model. Unclear what a user should make of it if the are in `main`